pkg/connector/group.go

```diff
@@ -101,7 +105,11 @@ func (g *groupBuilder) Entitlements(ctx context.Context, resource *v2.Resource,
 		Page:    pageToken,
 	})
 	if err != nil {
-		return nil, "", nil, err
+		if !strings.Contains(err.Error(), Unauthenticated) {
```

Comment:
You can check the status code on the error with something like:
```go
		if status.Code(err) != codes.Unauthenticated {
			...
		}
```

You'll need to import these:
```
	"google.golang.org/grpc/status"
	"google.golang.org/grpc/codes"
```

Reply:
Thanks for sharing the code. It looks nice and neat. However, in this case, status.Code(err) returns "Unknown" and codes.Unauthenticated = 16

Reply:
Fixed